wiki/useful-windows-tips.md

```diff
@@ -0,0 +1,42 @@
+# Useful tips and "hacks" for Windows
+
+### Get more options when right-clicking
+
+When right-clicking on a file or folder, you can hold down shift to get more options, such as to "Copy as path" which lets you copy the path of the file.
+
+
+### Windows Master Control Panel shortcut
+
+The Windows Master Control Panel shortcut, often referred to as "God Mode", is a shortcut to access all control panel settings in one place.
+
+To create the shortcut, create a folder and name it
+```
+allSettings.{ED7BA470-8E54-465E-825C-99712043E01C}
+```
+The text in front of the dot has no effect.
```

Comment:
"You may change it to whatever you want, but Windows will not show a name for the "folder""